data/json/items/armor.json

```diff
@@ -1005,7 +1005,7 @@
         "phase" : "solid",
         "encumbrance" : 5,
         "bashing" : -4,
-        "flags" : ["VARSIZE"],
+        "flags" : ["VARSIZE", "POCKETS"],
```

Comment:
With 5 warmth and 4 storage, it isn't really suited to have the POCKETS flag. Same for the two variants below.